client/src/modules/Opportunities/hooks/useExpiration.ts

```diff
@@ -1,11 +1,14 @@
-import moment from 'moment';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
 import { ExpirationState } from 'modules/Opportunities/constants';
 
+dayjs.extend(duration);
```

Comment:
Maybe we could extend `dayjs` globally somehow instead of doing it multiple times?